src/main/webapp/app/exam/participate/exam-navigation-bar/exam-navigation-bar.component.ts

```diff
@@ -89,6 +98,10 @@ export class ExamNavigationBarComponent implements OnInit {
         return this.exercises[this.exerciseIndex].type === ExerciseType.FILE_UPLOAD;
     }
 
+    setOverviewStatus(): 'active' | '' {
+        return this.overviewPageOpen ? 'active' : '';
+    }
+
```

Comment:
This doesn't set anything. get would be better in my opinion.

Reply:
done